deployment/ansible/roles/cac-tripplanner.app/tasks/jslibs.yml

```diff
@@ -11,7 +11,7 @@
        state=present
 
 - name: Install NPM dependencies
-  npm: path="{{ root_src_dir }}" state=present
+  command: npm install chdir="{{ root_src_dir }}"
```

Comment:
@CloudNiner If you can remember, this is why I asked you a while ago about the differences in meaning between `devDependencies` and `dependencies` in `package.json`. Ansible only really respects `dependencies`.